core/src/main/java/hudson/util/ListBoxModel.java

```diff
@@ -78,9 +78,9 @@
  * for producing option values. See the following example:
  *
  * <pre>
- * public ListBoxModel doOptionValues(@QueryParameter("value") String value) throws IOException, ServletException {
+ * public ListBoxModel doOptionValues(&#64;QueryParameter("value") String value) throws IOException, ServletException {
```

Comment:
I assume that a ```{@code``` wrapper would not work in this case because of the embedded ```@``` character?